src/bun.js/api/server.zig

```diff
@@ -183,6 +249,34 @@ pub const ServerConfig = struct {
                     any = true;
                 }
             }
+
+            if (obj.getTruthy(global, "key")) |js_obj| {
+                if (js_obj.jsType().isArray()) {
+                    const count = js_obj.getLengthOfArray(global);
+                    if (count > 0) {
+                        var native_array = bun.default_allocator.alloc([*c]const u8, count) catch unreachable;
+                        result.key = native_array.ptr;
+
+                        var i: u32 = 0;
+                        var valid_count: u32 = 0;
+
+                        while (i < count) : (i += 1) {
+                            var item = JSC.JSObject.getIndex(js_obj, global, i);
```

Comment:
```suggestion
                            const item = JSC.JSObject.getIndex(js_obj, global, i);
```

we should just add `getIndex` to `JSValue` I think. Then this becomes `js_obj.getIndex(global, i)`